docs/snippets/common/badge-story-starter-example.mdx.mdx

```diff
@@ -0,0 +1,51 @@
+```md
+
+<!--- Badge.stories.mdx --->
+
+<!--- This is your Story template function, shown here in React -->
```

Comment:
I think maybe you misunderstood my meaning in including these comments. I was referring to the MDX files where there is a single line:

```mdx
<!--- This is your Story template function, shown here in React -->
export const Template = (args) => <Button {...args} />

<Story name="X" args={{bla}} >
  {Template.bind({})}
</Story>
```

In such cases that line (`export const Template ...`) is the single thing that would be different between a react + angular version of the MDX snippet.

However in many of these examples there are large snippets of react-specific code and so I think they should "react" snippets. (Actually in this actual example I think we should maybe rewrite the snippet to use args, but in other examples they need to remain "react-ized").